### DISCUSSION TITLE
Is it safe to update sd-webui-controlnet?

### DISCUSSION THREAD

**Discussion by marcsyp** [Mikubill/sd-webui-controlnet#651] [General]
I pushed pause on updates last week while a lot of changes were happening.  Are the latest builds generally stable?  Anything major broken right now?  Thanks in advance!

**Comment by ljleb** [comment]
On my side I've been using the latest version in the last days and have not encountered any issue. I don't use all the features like compatibility with deforum, so I am not sure how well it works there at the moment.

You could clone a1111 webui + controlnet in a new folder to test whether it supports your use cases.
